test/controllers/IncomeControllerSpec.scala

```diff
@@ -315,41 +317,62 @@ class IncomeControllerSpec extends PlaySpec
 
   "updateEstimatedIncome" must {
     "return OK" when {
-      "income is successfully updated" in {
+
+      def expectedSuccessPage(employerName: String, employerId: Int,
```

Comment:
Is this being used?

Reply:
Nope it was old from when i was trying to run the specs a different way